phpspy.c

```diff
@@ -152,6 +152,37 @@ void usage(FILE *fp, int exit_code) {
     exit(exit_code);
 }
 
+static long strtol_with_min_or_exit(const char* name, const char* str, int min)
+{
+    long result;
+    char *end;
+    errno = 0;
+    result = strtol(str, &end, 10);
+    if (end <= str || *end != '\0') {
+        // e.g. reject -H '', -H invalid, -H 5000suffix
+        fprintf(stderr, "Expected integer for %s, got '%s'\n", name, str);
+        usage(stderr, 1);
+    }
+    if (result < min) {
+        // e.g. reject -H 0
```

Comment:
Can you make these C90 comments `/* ... */`?

Reply:
Done